### DISCUSSION TITLE
How to set the height of a column in Tableview?

### DISCUSSION THREAD

**Discussion by zhenzi0322** [israel-dryer/ttkbootstrap#346] [Q&A]
```python
import ttkbootstrap as ttk
from ttkbootstrap.tableview import Tableview
from ttkbootstrap.constants import *

app = ttk.Window()
colors = app.style.colors

coldata = [
    {"text": "LicenseNumber", "stretch": False},
    "CompanyName",
    {"text": "UserCount", "stretch": False},
]

rowdata = [
    ('A123', 'IzzyCo', 12),
    ('A136', 'Kimdee Inc.', 45),
    ('A158', 'Farmadding Co.', 36)
]

dt = Tableview(
    master=app,
    coldata=coldata,
    rowdata=rowdata,
    paginated=True,
    searchable=True,
    height=20,  # Tableview height
    bootstyle=PRIMARY,
    stripecolor=(colors.light, None),
)
dt.pack(fill=BOTH, expand=YES, padx=10, pady=10)

app.mainloop()
```

**Comment by antrrax** [comment]
Changing the height of a single row is not possible, but you can create a custom style to change the height of all rows

```python
import ttkbootstrap as ttk
from ttkbootstrap.tableview import Tableview
from ttkbootstrap.constants import *

app = ttk.Window()
colors = app.style.colors

style = ttk.Style()
style.configure('custom.primary.Treeview', rowheight=100, font=('', 12))
style.configure('custom.primary.Treeview.Heading', font=('', 10, 'bold'))

coldata = [
    {"text": "LicenseNumber", "stretch": False},
    "CompanyName",
    {"text": "UserCount", "stretch": False},
]

rowdata = [
    ('A123', 'IzzyCo', 12),
    ('A136', 'Kimdee Inc.', 45),
    ('A158', 'Farmadding Co.', 36)
]

dt = Tableview(
    master=app,
    coldata=coldata,
    rowdata=rowdata,
    paginated=True,
    searchable=True,
    height=20,  # Tableview height
    bootstyle=PRIMARY,
    stripecolor=(colors.light, None),
)
dt.pack(fill=BOTH, expand=YES, padx=10, pady=10)

dt.configure(style='custom.primary.Treeview')

app.mainloop()
```

see too:
https://github.com/israel-dryer/ttkbootstrap/discussions/313

**Reply by zhenzi0322** [reply]
> Changing the height of a single row is not possible, but you can create a custom style to change the height of all rows
> 
> ```python
> import ttkbootstrap as ttk
> from ttkbootstrap.tableview import Tableview
> from ttkbootstrap.constants import *
> 
> app = ttk.Window()
> colors = app.style.colors
> 
> style = ttk.Style()
> style.configure('custom.primary.Treeview', rowheight=100, font=('', 12))
> style.configure('custom.primary.Treeview.Heading', font=('', 10, 'bold'))
> 
> coldata = [
>     {"text": "LicenseNumber", "stretch": False},
>     "CompanyName",
>     {"text": "UserCount", "stretch": False},
> ]
> 
> rowdata = [
>     ('A123', 'IzzyCo', 12),
>     ('A136', 'Kimdee Inc.', 45),
>     ('A158', 'Farmadding Co.', 36)
> ]
> 
> dt = Tableview(
>     master=app,
>     coldata=coldata,
>     rowdata=rowdata,
>     paginated=True,
>     searchable=True,
>     height=20,  # Tableview height
>     bootstyle=PRIMARY,
>     stripecolor=(colors.light, None),
> )
> dt.pack(fill=BOTH, expand=YES, padx=10, pady=10)
> 
> dt.configure(style='custom.primary.Treeview')
> 
> app.mainloop()
> ```
> 
> see too: #313

ok, Thank you.
